zerver/views/auth.py

```diff
@@ -330,6 +330,8 @@ def login_page(request, **kwargs):
     if is_subdomain_root_or_alias(request) and settings.REALMS_HAVE_SUBDOMAINS:
         redirect_url = reverse('zerver.views.registration.find_my_team')
         return HttpResponseRedirect(redirect_url)
+    if request.user.is_authenticated():
```

Comment:
Do we want this earlier in the function, before the subdomain stuff?  If you don't know the answer, perhaps @timabbott can weight in.  (I don't know the subdomain-related issues very well.)